api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java

```diff
@@ -160,11 +190,31 @@
                 .limit(PAGE_SIZE)
                 .collect(Collectors.toList());
 
+        stopWatch = new StopWatch();
+        stopWatch.start();
+        log.log(Level.INFO, "starting saveCohortReview()");
         cohortReviewService.saveCohortReview(cohortReview);
+        stopWatch.stop();
+        log.log(Level.INFO, "saveCohortReview() time:" + stopWatch.getTotalTimeSeconds());
+        stopWatch = new StopWatch();
+        stopWatch.start();
+        log.log(Level.INFO, "starting saveParticipantCohortStatuses()");
         cohortReviewService.saveParticipantCohortStatuses(participantCohortStatuses);
```

Comment:
Just a note: doing this in a separate transaction from saveCohortReview() means we could wind up in a situation where we have a review with no participant cohort statuses in it. 

Once we figure out how to address the performance bottleneck, we may need to figure out how to either write the status rows in the same transaction or handle this case gracefully.

Reply:
Agreed.